ray/README.md

```diff
@@ -1,68 +1,83 @@
-# Deploying Ray with Open Data Hub
+# Deploying Ray on Open Data Hub via the KubeRay Operator
 
-Integration of [Ray](https://docs.ray.io/en/latest/index.html) with Open Data Hub on OpenShift. The ray operator and other components are based on https://docs.ray.io/en/releases-1.13.0/cluster/kubernetes.html
+The code found here is a subset of https://github.com/ray-project/kuberay. Specifically, the manifests needed to enable KubeRay with an Open Data Hub deployment. 
 
-## Components  of the Ray deployment
 
-1. [Ray operator](./operator/base/ray-operator-deployment.yaml): The operator will process RayCluster resources and schedule ray head and worker pods based on requirements. Image Containerfile can be found [here](https://github.com/thoth-station/ray-operator/blob/master/Containerfile). 
-2. [Ray CR](./operator/base/ray-custom-resources.yaml):  RayCluster Custom Resource (CR) describes the desired state of ray cluster.
-3. [Ray Cluster](./cluster/base/ray-cluster.yaml): Defines an instance of an example Ray Cluster. Image Containerfile can be found [here](https://github.com/thoth-station/ray-ml-worker/blob/master/Containerfile) 
- 
+## Components of the KubeRay deployment
 
-## Deploy the RayCluster Components:
+1. Namespace 'ray-system'
+2. Custom Resource Definitions
+3. Operator
+4. Roles and Service Accounts
 
-Prerequisite to install RayCluster with ODH:
+## Installation
 
-* Cluster admin access
-* An ODH deployment
-* [Kustomize](https://kustomize.io/) 
+There are two ways to install KubeRay. You can either **install it directly into your cluster alongside an existing Open Data Hub deployment using Kustomize** OR you can **include it as an additional component in your Open Data Hub KfDef**. Below we will walk through both approaches. 
 
-### Install Ray
 
-We will use [Kustomize](https://kustomize.io/) to deploy everything we need to use Ray with Open Data Hub. 
+## Install via Kustomize:
 
-#### Install the operator and custom resource 
-
-First use the `oc kustomize` command to generate a yaml containing all the requirements for the operator and the "raycluster" custom resource, then `oc apply` that yaml to deploy the operator to your cluster. 
+This method requires that  to have [kustomize](https://kustomize.io/) and the [oc client](https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html) installed in your environment. Then, from within the `ray/` directory of this repo you can run the following:
```

Comment:
You have a typo here "requires that  to have". I think this should be something like "requires that you have"